packages/test/end-to-end-tests/src/test/loaderTest.spec.ts

```diff
@@ -218,4 +233,12 @@ describe("Loader.request", () => {
         assert.strictEqual(sameDataStore1, sameDataStore2,
             "same containers do not return same data store for same request");
     });
+    it("request can handle url with query params", async () => {
+        const url = await container.getAbsoluteUrl("");
+
+        const query = `?query1=1&query2=2`;
+        const testUrl = `${url}${query}`;
+        const response = await loader.request({ url: testUrl });
+        assert.strictEqual(response.value, query, "request did not pass the right query");
```

Comment:
nit: "request did not pass the right query to the data store" may be a little better description.